TVOSExample/metro.config.js

```diff
@@ -51,12 +58,6 @@ const config = {
     // to various errors. To mitigate this we define below custom request resolver, hijacking requests to conflicting modules and manually
     // resolving appropriate files. **Most likely** this can be achieved by proper usage of blockList but I found this method working ¯\_(ツ)_/¯
     resolveRequest: (context, moduleName, platform) => {
-      if (moduleName.startsWith('@react-navigation')) {
-        // For some reason, react-navigation packages don't want to resolve from
-        // the project's node_modules, so we need to use standard Metro resolver.
-        return context.resolveRequest(context, moduleName, platform);
-      }
```

Comment:
Uuuuu, why do we remove this? Have you run TVOS example to confirm it does not fail in runtime?

Reply:
The TVOS example failed in runtime failed at first. I can confirm it works with these changes, which are in fact copied from the Example app